safe/impact_functions/inundation/flood_polygon_population/parameter_definitions.py

```diff
@@ -45,7 +65,11 @@ def affected_field():
 
 
 def affected_value():
-    """Generator for parameter stating what values constitute 'affected'."""
+    """Generate affected value parameter
+
+    :return: list of String Parameter
+    :rtype: list[StringParameter]
```

Comment:
I think this should just be list - please check in pycharm if it correctly interprets the rtype if you specify a type in the list.